clients/vault/src/oracle/storage/impls.rs

```diff
@@ -427,7 +427,7 @@ mod test {
 			.expect("should have an archive url")
 			.clone();
 		let scp_archive_storage = ScpArchiveStorage(testnet_archive_url);
-		let slot_index = 235001;
```

Comment:
How about we use an even smaller value? Maybe even slot 0? Then we are 100% resilient for future testnet resets.